examples/src/main/scala/zio/config/examples/ProgramExample.scala

```diff
@@ -66,69 +35,65 @@ object ProgramExample extends App {
   }
 }
 
-// bootstrap modules
-object bootstrap {
-  type SparkSession = String
-
-  trait SparkEnv {
-    def spark: SparkEnv.Service
-  }
-
-  object SparkEnv {
-    type SparkSession = String
-
-    trait Service {
-      def getSpark: Task[SparkSession]
-    }
-  }
-
-  case class Bootstrap(configService: ConfigSource.Service, spark: SparkEnv.Service) extends SparkEnv with ConfigSource
+// spark service
+trait SparkEnv {
+  def spark: SparkEnv.Service
 }
 
-// client modules
-object client {
-
-  // Let sparksession be string for example simplicity.
+object SparkEnv {
   type SparkSession = String
 
-  case class Client(sparkSession: SparkSession, config: ProgramConfig) extends Spark with Config
-
-  trait Spark {
-    val sparkSession: SparkSession
+  trait Service {
+    def getSpark: UIO[SparkSession]
   }
 
-  trait Config {
-    val config: ProgramConfig
-  }
+  def make: IO[Throwable, SparkEnv] =
+    ZIO
+      .effect("sparkEnv")
+      .map(
+        r =>
+          new SparkEnv {
+            override def spark: Service = new Service {
```

Comment:
you can define in  the object `SparkEnv`
```scala
 object Test extends Service {
  def getSpark: UIO[SparkSession] = ZIO.succeed(r)
}
```
then you can use it:
```scala
def make: Task[SparkEnv] = Task(SparkEnv.Test)
```
(to not confuse you you can use IO[Throwable, SparkEnv], Task is just a type alias for IOs that fail with throwable and if we call `Task(???)` this is the same as `IO.effect(...)`